llama_hub/library.json

```diff
@@ -955,5 +955,14 @@
       "repository",
       "collaborators"
     ]
+  },
+  "NougatOCRReader": {
+    "id": "github_repo_collaborators",
```

Comment:
id is duplicated

Reply:
Great, thanks.  I'll fix and push it asap.